### ISSUE TITLE
Flunks go vet

### ISSUE THREAD

**Issue by ryboe** [tmthrgd/go-bindata#4]
```sh
$ go vet $(go list ./... | grep -v /vendor/)  # avoid scanning vendor/
header.go:71: +build comment must appear before package clause and be followed by a blank line
```

**Comment by tmthrgd** [comment]
This is actually a bug in go vet not in go-bindata. That build comment is inside a string literal (see [header.go:71](https://github.com/tmthrgd/go-bindata/blob/eea405957029908d3e235f8d1c8749213bd1985f/header.go#L71)) and is part of the generated output.

While it is possible to silence this warning with a hack, I'd much rather not. I'll leave this open and marked wontfix, and I'll submit a bug report for go vet and see if that can be fixed.

**Comment by tmthrgd** [comment]
The go vet bug has already been reported in golang/go#12269 and golang/go#13533. The first was closed with the given reason "Not worth the trouble to fix." The second has also been closed with the given reason "probably not worth doing".

Unless you have a particularly pressing reason to address this, I think I'd rather leave it as wontfix.

**Comment by tmthrgd** [comment]
I reneged, and I fixed it in d825d59 with a small workaround.
